source/particle/interpolator/bilinear_least_squares.cc

```diff
@@ -51,6 +54,18 @@ namespace aspect
         AssertThrow(selected_properties.n_selected_components(n_particle_properties) == 1,
                     ExcNotImplemented("Interpolation of multiple components is not supported."));
 
+        if (use_global_valued_limter)
+          {
+            AssertThrow(global_minimum_particle_properties.size() == n_particle_properties,
+                        ExcMessage("Make sure that the size of 'Global minimum particle property' is equivalent to the number "
+                                   "of particle properties."));
+
+            AssertThrow(global_maximum_particle_properties.size() == n_particle_properties,
+                        ExcMessage("Make sure that the size of 'Global maximum particle property' is equivalent to the number "
+                                   "of particle properties."));
+
+          }
```

Comment:
In calling the function
`this->template find_postprocessor<Postprocess::Particles<dim> >();` from `aspect::Particle::Interpolator::BilinearLeastSquares<dim>::properties_at_points()`, my compiler produces the error message

```
/home/harsha_lv/ClionProjects/aspect/include/aspect/simulator_access.h: In instantiation of ‘PostprocessorType* aspect::SimulatorAccess<dim>::find_postprocessor() const [with PostprocessorType = aspect::Postprocess::Particles<2>; int dim = 2]’:
/home/harsha_lv/ClionProjects/aspect/source/particle/interpolator/bilinear_least_squares.cc:44:135:   required from ‘std::vector<std::vector<double> > aspect::Particle::Interpolator::BilinearLeastSquares<dim>::properties_at_points(const std::multimap<std::pair<int, int>, aspect::Particle::Particle<dim> >&, const std::vector<dealii::Point<dim> >&, const dealii::ComponentMask&, const typename dealii::parallel::distributed::Triangulation<dim>::active_cell_iterator&) const [with int dim = 2; typename dealii::parallel::distributed::Triangulation<dim>::active_cell_iterator = dealii::TriaActiveIterator<dealii::CellAccessor<2, 2> >]’
/home/harsha_lv/ClionProjects/aspect/source/particle/interpolator/bilinear_least_squares.cc:255:7:   required from here
/home/harsha_lv/ClionProjects/aspect/include/aspect/simulator_access.h:684:90: error: invalid use of incomplete type ‘const class aspect::Simulator<2>’
     return simulator->postprocess_manager.template find_postprocessor<PostprocessorType>();

```
The BilinearLeastSquares class inherits from the Particle::Interpolator::Interface class and SimulatorAccess class. Does one need to treat the registration of the BilinearLeastSquares class in a special way to use the function `find_postprocessor`?? If you have any quick leads, that would be helpful. Thanks.